src/components/FlowRunTimelineSubFlowRunDetails.vue

```diff
@@ -43,18 +40,50 @@
   const alternate = true
 
   const routes = useWorkspaceRoutes()
-
   const api = useWorkspaceApi()
+
   const flowRunSubscriptionArgs = computed<[string]>(() => [flowRunId.value])
   const flowRunSubscription = useSubscriptionWithDependencies(
     api.flowRuns.getFlowRun,
     flowRunSubscriptionArgs,
   )
 
   const subFlowRun = computed(() => flowRunSubscription.response)
+
+  const flowSubscriptionArgs = computed<[string]>(() => [flowRunSubscription.response?.flowId || ''])
+  const flowSubscription = useSubscriptionWithDependencies(
+    api.flows.getFlow,
+    flowSubscriptionArgs,
+  )
+
+  const flow = computed(() => flowSubscription.response)
```

Comment:
Same here
```
const flowId = computed(() => subFlowRun.value?.flowId)
const { flow } = useFlow(flowId)
```

Reply:
Maybe we can look at this together when we pair. When another node is selected, the component doesn't remount, it get's a new ID. So here it'll actually update the data, but there's probably another way.

Reply:
The id is reactive. So if flowId changes it'll fetch the new flow.